systemtests/pom.xml

```diff
@@ -14,6 +14,7 @@
         <!-- System test image dependencies -->
         <keycloak.image>quay.io/keycloak/keycloak:14.0.0</keycloak.image>
         <strimzi-kafka.tag>quay.io/strimzi/kafka:0.26.0-kafka-3.0.0</strimzi-kafka.tag>
+        <jacoco.skip>true</jacoco.skip>
     </properties>
```

Comment:
I don't think we should skip Jacoco completely, but rather skip the classes/packages in the systemtests project. We'll want to gather the coverage of the `kafka-admin` module for the tests executed in systemtests.

https://www.baeldung.com/jacoco-report-exclude

Reply:
So, when using the exclude parameter, the rule fails for the systemtests. It returns a 0.0 for the line coverage.

Reply:
> rather skip the classes/packages in the systemtests project.

I couldn't exclude it from `<rule>` but I was able to skip the above.